src/utils/formatter.ts

```diff
@@ -1,21 +1,21 @@
-import { getCardTypeByValue, DEFAULT_CARD_FORMAT } from "./cardTypes";
+import { getCardTypeByValue, DEFAULT_CARD_FORMAT } from './cardTypes';
 
 export const formatCardNumber = (cardNumber: string) => {
   // 1. get card type
   const cardType = getCardTypeByValue(cardNumber);
 
   // 2. format cardNumber by cardType format
   if (cardType) {
-    return (cardNumber.match(cardType.format) || []).join(" ");
+    return (cardNumber.match(cardType.format) || []).join(' ');
   }
 
-  return (cardNumber.match(DEFAULT_CARD_FORMAT) || []).join(" ");
+  return (cardNumber.match(DEFAULT_CARD_FORMAT) || []).join(' ');
 };
 
 export const formatExpiry = (event: React.ChangeEvent<HTMLInputElement>) => {
   // @ts-ignore
```

Comment:
see the issue here, I'll go through and refactor my solution, so we don't have to `ts-ignore` anything..